docs/more/external-libs.md

```diff
@@ -1,27 +1,31 @@
 ---
 title: External libraries
-description: Libraries that can help with yew development
+description: Libraries that can help with Yew development
 ---
 
 ### Yewtil
 
-Yewtil is a collection of common utilities that help you write Yew programs. It includes:
+Yewtil is a collection of common utilities that help you build applications using Yew. It includes:
 
-* NeqAssign - As discussed earlier, is the best way to assign props to ensure minimal re-rendering.
-* PureComponents - Components that don't update any of their state. Using NeqAssign under the hood, they act as memoized 
+* NeqAssign - This is described in more detail in the section on 
+[optimisations and best practices](../advanced-topics/optimizations.md) and ensures that identical
+sets of props don't cause a component to re-render.
 
+* PureComponents - Components that don't update any of their state. Using NeqAssign under the hood, they act as memoized 
   functions that are called from inside the `html!` macro like normal components are.
 
 * Lrc - linked list reference counted smart pointer functions like `Rc` does, but allows for novel data update patterns.
 * Mrc/Irc - Mutable/Immutable reference counted smart pointers that function like `Rc` but are more ergonomic to use 
 
-  within Yew, due to implementing `DerefMut` and `BorrowMut`for `Mrc`. This allows `Mrc` to be used with `NeqAssign`. 
+  within Yew, because they implement `DerefMut` and `BorrowMut`for `Mrc`. This allows `Mrc` to be used with `NeqAssign`. 
 
-  `Irc` acts as an immutable view into the data, which makes this ideal for holding data used in display-only tasks.
+  `Irc` acts as an immutable view of the data, which makes this ideal for holding data used in display-only tasks.
 
-* History - A history tracking wrapper that uses a `VecDeque` to hold on to previous values that it has represented.
+* History - A history tracking wrapper that uses a `VecDeque` to hold on to previous values that it 
+has represented.
 * Futures - Support for running futures that send messages to component update loops.
-* Fetch - Abstractions for handling fetch requests made using `web_sys` and the aforementioned futures feature.
+* Fetch - Abstractions for handling fetch requests made using `web_sys` in conjunction with the  
+futures feature.
```

Comment:
```suggestion
"futures" feature.
```

Reply:
Not actually sure if that's what the text is going for... Do we even need the "in conjunction with the futures feature."?

Reply:
I'm not exactly sure either. The stuff about the futures feature can probably go away.

Reply:
I vote for it to be removed :)